decidim-debates/spec/system/show_spec.rb

```diff
@@ -6,13 +6,56 @@
   include_context "with a component"
   let(:manifest_name) { "debates" }
 
-  let!(:debate) { create(:debate, component: component, skip_injection: true) }
+  let(:description) { Decidim::Faker::Localized.wrapped("<p>", "</p>") { generate_localized_debate_title } }
+  let(:information_updates) { Decidim::Faker::Localized.wrapped("<p>", "</p>") { generate_localized_debate_title } }
+  let(:instructions) { Decidim::Faker::Localized.wrapped("<p>", "</p>") { generate_localized_debate_title } }
+  let!(:debate) { create(:debate, component: component, description: description, information_updates: information_updates, instructions: instructions, skip_injection: true) }
 
   before do
     visit_component
     click_link debate.title[I18n.locale.to_s], class: "card__link"
   end
 
+  context "when is created from the admin panel" do
+    let!(:debate) { create(:debate, :official, component: component, description: description, information_updates: information_updates, instructions: instructions) }
+
+    context "when the field is decription" do
+      it_behaves_like "has embedded video in description", :description
+    end
+
+    context "when the field is information_updates" do
+      it_behaves_like "has embedded video in description", :information_updates
+    end
+
+    context "when the field is instructions" do
+      it_behaves_like "has embedded video in description", :instructions
+    end
+  end
+
+  context "when is created by the participant" do
+    let!(:author) { create(:user, :confirmed, organization: component.organization) }
+    let!(:debate) { create(:debate, author: author, component: component, description: description, information_updates: information_updates, instructions: instructions) }
```

Comment:
I think you could use here `:citizen_author` instead of `:participant_author` in the original PR. This trait was changed at #8697.
```suggestion
    let!(:debate) { create(:debate, :citizen_author, component: component, description: description, information_updates: information_updates, instructions: instructions) }
```